variables.tf

```diff
@@ -25,7 +37,7 @@ variable "display_name" {
 variable "policy" {
   description = "The fully-formed AWS policy as JSON"
   type        = string
-  default     = null
+  default     = ""
```

Comment:
Keep it `null` since policy is optional.

Reply:
Roger that, done!